src/debugAdapter/goDebug.ts

```diff
@@ -321,21 +321,36 @@ class GoDebugSession extends DebugSession {
 		log('InitializeEvent');
 	}
 
+	protected findPathSeperator(path) {
+		if (/^(\w:[\\/]|\\\\)/.test(path)) return '\\';
```

Comment:
It would just be simpler. But I guess if the path is like `C:/foo/bar` you still want to return a backslash